package.json

```diff
@@ -129,12 +128,24 @@
       "git add"
     ]
   },
+  "husky": {
```

Comment:
`husky` was complaining that these hooks were in run scripts, so moved to this husky config block